harness/determined/experimental/_native.py

```diff
@@ -194,17 +182,18 @@ def make_local_experiment_config(input_config: Optional[Dict[str, Any]]) -> Dict
     for key in config_keys_to_ignore:
         if key in input_config:
             logging.info(
-                f"'{key}' configuration key is not supported by LOCAL mode and will be ignored"
+                f"'{key}' configuration key is not supported by local=True,test=True mode and will"
```

Comment:
Just "local test mode"? This "local=True,test=True" seems a bit awkward.

Reply:
sure